moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1974,61 +2026,59 @@ double RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<
 
   double percentage = 1.0;
   // compute the average distance between the states we looked at
-  double thres = jump_threshold * (total_dist / (double)dist_vector.size());
+  double thres = jump_threshold_factor * (total_dist / (double)dist_vector.size());
   for (std::size_t i = 0; i < dist_vector.size(); ++i)
     if (dist_vector[i] > thres)
     {
       CONSOLE_BRIDGE_logDebug("Truncating Cartesian path due to detected jump in joint-space distance");
-      percentage = (double)i / (double)dist_vector.size();
-      traj.resize(i);
+      percentage = (double)(i + 1) / (double)(traj.size());
```

Comment:
Looks like you identified another bug. Well done!

Reply:
thanks